migrations-1/002_marketplace1.up.sql

```diff
@@ -0,0 +1,192 @@
+
+-- Run marketplace2.sql in marketplace2 container first , then run this file
+-- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+CREATE EXTENSION postgres_fdw;
+CREATE SERVER node1 FOREIGN DATA WRAPPER postgres_fdw OPTIONS (host 'marketplace2', dbname 'ds_db');
+CREATE USER MAPPING FOR postgres SERVER node1 OPTIONS (user 'postgres' , password '1700455');
+
+
+-- User table
+
+CREATE TABLE "users" ("id" bigserial,"email" text,"password" text,"name" text,"balance" decimal,"image_url" text,"created_at" timestamptz,"updated_at" timestamptz,"deleted_at" timestamptz);
+CREATE INDEX "idx_users_deleted_at" ON "users" ("deleted_at");
+CREATE INDEX "idx_users_id" ON "users" ("id");
+
```

Comment:
```suggestion
CREATE INDEX "idx_users_email" ON "users" ("email");
```
or if we can make unique indexes 
```suggestion
CREATE UNIQUE INDEX "unique_idx_users_email" ON "users" ("email");
```
actually, if we can make unique indexes we should make all indexes on ids unique